### PR TITLE
Created first draft of documentation for umbrella charts

### PR DESCRIPTION
**Pull request by jaxels10**
This is a first draft and is (probably) subject to a lot of changes. Please comment if you think something is vague, missing or redundant.

### CODE REVIEW

File: git/umbrella-charts.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- Problem 4: We would like to test our Helm charts to make sure that both current and new versions will have the correct functionality needed. 
+
+## Discussion
+The team had a discussion on the best way to try and either solve these problems or at least find a feasible solution that will at least work temporarily. We had multiple suggestions in play, one being to create a single repository to hold all custom manifests and resources. However, during the discussion, our ideas seemed to add more complexity and change the way Yggdrasil would work. After some discussion we ended up on the concept that we already had been using before which was umbrella charts. 
+
+## Umbrella charts
+An umbrella chart is a chart that holds another chart(s) as a dependency. This enables us to put custom manifests and resources into the templates folder of the umbrella chart, as well as having a protected main branch and good branching conventions. This means that we will add a repository for every service we have running on the cluster, even if we don't initially configure it.
```
Comment (skovbakke) [inline git/umbrella-charts.md:22]: Mention that it is a place for us to put acceptance tests.

**Review by skovbakke — COMMENTED** [review]
I personally think it is too wordy. To me the first four paragraphs are not valuable. But maybe @MartinSchmidt has an opinion on this.

File: git/umbrella-charts.md (hunk 1 above)
Comment (jaxels10) [reply git/umbrella-charts.md]: I put it in there now :)

**Comment by jacobcrawford** [comment]
I remember a discussion on the matter, where we would like to be able to disable a dependency without disabling the whole chart. Maybe we should require an "enabled" field in the values file for each dependency. This should fit into a "how to" section, which I think is missing in the wiki, but described as an acceptance criterion in Jira.

**Comment by jaxels10** [comment]
> I remember a discussion on the matter, where we would like to be able to disable a dependency without disabling the whole chart. Maybe we should require an "enabled" field in the values file for each dependency. This should fit into a "how to" section, which I think is missing in the wiki, but described as an acceptance criterion in Jira.

Did we agree to have an install condition for every dependency in the umbrella chart, or was this just a talk? I can add it to the documentation if needed. Also, I am adding an "how to" section right now.

**Comment by jaxels10** [comment]
> I personally think it is too wordy. To me the first four paragraphs are not valuable. But maybe @MartinSchmidt has an opinion on this.

I am fine with not including the paragraphs about the discussion. Perhaps @MartinSchmidt can give us his opinion.

**Comment by jaxels10** [comment]
I have added a "how-to" section to the documentation.

**Review by MartinSchmidt — APPROVED** [review]

**Review by skovbakke — APPROVED** [review]
